content/en/threat-model.md

```diff
@@ -0,0 +1,141 @@
+|                        |                  | x             |                       | x            |               |              |             | Same as Fulcio CT log + Rekor server, but can now use the replayed certs created by Fulcio CT log because Rekor will not report that they are already in the log, even with more sophisticated monitors.                                                                                                                                                                                                                                                                                                                                                     |
+|                        |                  |               |                       | x            | x             |              |             | Same as Rekor server, but changes to the CT log would not be reported by the monitor, would make recovery of the log much harder because the attacker could alter the historical entries, may require re-signing all metadata as previous timestamps are untrusted.                                                                                                                                                                                                                                                                                      |
+|                        |                  | x             | x                     | x            |               |              |             | Same as [Fulcio CT log + Rekor server] | Replay and fork attacks will not be reported by monitors.                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+|                        | x                |               |                       |              |               |              |             | Arbitrary software attack: Can issue certs for any OIDC issuer/identity and use those to sign any software desired. Bob will see these certificates as valid as they are signed by the Fulcio CA and included in the Fulcio CT log. | Fulcio CT log prevents changes to the history  or blocking issuance / modification of past things (which is not that relevant given the ability to issue keys, etc. now) To recover, the TUF root of trust for Fulcio needs to be invoked to revoke trust in the compromised Fulcio CA server.    |
```

Comment:
For slashes, I would be consistent on whether there are spaces or not. What is meant by "things" in "past things"? 

```suggestion
|                        | x                |               |                       |              |               |              |             | Arbitrary software attack: Can issue certs for any OIDC issuer/identity and use those to sign any software desired. Bob will see these certificates as valid as they are signed by the Fulcio CA and included in the Fulcio CT log. | Fulcio CT log prevents changes to the history  or blocking issuance/modification of past things (which is not that relevant given the ability to issue keys, etc., now) To recover, the TUF root of trust for Fulcio needs to be invoked to revoke trust in the compromised Fulcio CA server.    |
```